bindings/experimental/distrdf/python/DistRDF/Backends/Base.py

```diff
@@ -159,7 +159,16 @@ def mapper(current_range):
                 list: This respresents the list of (mergeable)values of all
                 action nodes in the computational graph.
             """
-            import ROOT
+            # - Disable graphics functionality in ROOT. It is not needed inside
+            #   a distributed task
+            # - Enable thread safety for the whole mapper function. There are
```

Comment:
`- We need to enable thread safety since two tasks could be invoking the C++ interpreter simultaneously, given that mapper tasks release the GIL before calling into C++ to run the event loop. Dask multi-threaded or even multi-process workers could trigger such a scenario.`